cmd/gh/main.go

```diff
@@ -354,24 +355,22 @@ func checkForUpdate(currentVersion string) (*update.ReleaseInfo, error) {
 // BasicClient returns an API client for github.com only that borrows from but
 // does not depend on user configuration
 func basicClient(currentVersion string) (*api.Client, error) {
-	var opts []api.ClientOption
-	if isVerbose, debugValue := utils.IsDebugEnabled(); isVerbose {
-		colorize := utils.IsTerminal(os.Stderr)
-		logTraffic := strings.Contains(debugValue, "api")
-		opts = append(opts, api.VerboseLog(colorable.NewColorable(os.Stderr), logTraffic, colorize))
+	var log io.Writer
+	if debugEnabled, _ := utils.IsDebugEnabled(); debugEnabled {
```

Comment:
`go-gh` only provides one level of logging so if there is any debug environment variable present then turn on logging.